### ISSUE TITLE
Call of Eris DB method 'getNetworkInfo' with parameters undefined responded with "".

### ISSUE THREAD

**Issue by jaeseokan94** [eris-ltd/eris-db.js#49]
Hi, 

I followed the tutorial from Eris website, and following is my codes. I am trying to get network and blockchain info from the Eris-db server running on my laptop. On the website, no parameter is required for this two methods, but when I call this, it responded with "".  What is went wrong and what should I check?  

    var edbFactory = require('eris-db');
    var edb = edbFactory.createInstance("http://172.17.0.3:1337/rpc");

    edb.network().getInfo(function(error,data){
    console.log("Getting info successfully");
    console.log(error);
    });
    edb.blockchain().getInfo(function(error,data){
     console.log(error);   
    });

Error message shows something like 

{ Error
    at values.next.then (/Users/JAESEOKAN/node_modules/eris-db/node_modules/@nodeguy/json-rpc/lib/index.js:12:38)
  method: 'getNetworkInfo',
  params: [],
  code: 'ECONNREFUSED',
  errno: 'ECONNREFUSED',
  syscall: 'connect',
  address: '172.17.0.3',
  port: 1337 }

Thanks,
Jae

**Comment by jaeseokan94** [comment]
INFO[02-16|17:46:23] Starting EventSwitch                     module=events impl=EventSwitch
NOTE[02-16|17:46:23] Loaded PrivValidator                     module=types file=/users/jaeseokan/.eris/chains/simplechain/simplechain_full_000/priv_validator.json privValidator="PrivValidator{1CF0F3C9695568D16B2590C7C0C3D375D5497B89 LH:8, LR:0, LS:3}"
Force disabling Tendermint's native RPC, which had been set to run on '0.0.0.0:46657' in the Tendermint config.
INFO[02-16|17:46:23] Starting EventSwitch                     module=events impl=EventSwitch
DBUG[02-16|17:46:23] addRound(round)                          module=consensus round=0
INFO[02-16|17:46:23] Local listener                           module=p2p ip=:: port=46656
INFO[02-16|17:46:23] Getting UPNP external address            module=p2p
^C


This is what I found from the log when I start running Eris-db server.

**Comment by jaeseokan94** [comment]
I have obtained the ip address of eris-db server running on my chain using following command as suggested on tutorial. "eris chains inspect <name of ErisDB server> NetworkSettings.IPAddress" and I got the value of 172.17.0.3.  It gives ECONNREFUSED error. 

I got the IP address of docker-machine using this command. "docker-machine ip eris", and it gives 192.168.99.100 value. 
My app starts to work after changing IP address to 192.168.99.100.  

var edbFactory = require('eris-db');
var edb = edbFactory.createInstance("http://192.168.99.100:1337/rpc");
